internal/expr/prepare.go

```diff
@@ -146,10 +153,10 @@ func prepareOutput(ti typeNameToInfo, p *outputPart) ([]fullName, []typeMember,
 			if t.name == "*" {
 				switch info := info.(type) {
 				case *mapInfo:
-					return nil, nil, fmt.Errorf(`&%s.* cannot be used for maps when no column names are specified`, info.typ().Name())
+					return nil, nil, fmt.Errorf(`columns must be specified for asterisk map`)
 				case *structInfo:
 					if len(info.tags) == 0 {
-						return nil, nil, fmt.Errorf("type %q in %q does not have any db tags", info.typ().Name(), p.raw)
+						return nil, nil, fmt.Errorf("no db tags found for type %q", info.typ().Name())
```

Comment:
Perhaps more emphasis should be placed on 'db' and the fact that these are struct fields could be highlighted? I'm thinking just to make sure it's Go-tags that the person reading this thinks of, not some other unrelated database tags. Would something like this work?
```
`no "db:" tags found on struct %q`
```

Reply:
Sounds like a good idea, changed.